### ISSUE TITLE
Unable to deploy v1.6.4 and v1.7.0-beta.0 using Kubernetes-Anywhere v0.1.0

### ISSUE THREAD

**Issue by divyenpatel** [vmware/kubernetes#151]
Tried deploying Kubernetes Cluster using v1.6.4 and v1.7.0-beta.0 release and observed nodes status remains pending.

Used Kubernetes-Anywhere for the deployment.

Logs
-----
```
root@master [ ~ ]# journalctl -u kubelet.service -f
.
.

Jun 07 21:52:52 master docker[1628]: Error: unknown flag: --config
Jun 07 21:53:12 master systemd[1]: kubelet.service: Main process exited, code=exited, status=1/FAILURE
Jun 07 21:53:12 master systemd[1]: kubelet.service: Unit entered failed state.
Jun 07 21:53:12 master systemd[1]: kubelet.service: Failed with result 'exit-code'.
Jun 07 21:53:22 master systemd[1]: kubelet.service: Service hold-off time over, scheduling restart.
Jun 07 21:53:22 master systemd[1]: Stopped Kubernetes Kubelet Server.
Jun 07 21:53:22 master systemd[1]: Starting Kubernetes Kubelet Server...
```

```
make[1]: Leaving directory '/opt/kubernetes-anywhere'
KUBECONFIG="$(pwd)/phase1/$(jq -r '.phase1.cloud_provider' .config.json)/.tmp/kubeconfig.json" ./util/validate
Validation: Expected 5 healthy nodes; found 0. (280s elapsed)
Validation: Expected 5 healthy nodes; found 0. (290s elapsed)
.
.
Validation: Expected 5 healthy nodes; found 0. (580s elapsed)
.
Validation: Expected 5 healthy nodes; found 0. (730s elapsed)
```

cc: @tusharnt @BaluDontu 

This is P0 and We need to investigate, the deployment issue.
CI jobs picks up latest bits from the release 1.6 and there we never observed deployment issue.

**Comment by divyenpatel** [comment]
Ignition Image - `docker.io/cnastorage/k8s-ignition:v1` uses kubelet flag "`--config`", which is no longer available on v1.6.4 and v1.7.0-beta.0. We need to use  new flag "`--pod-manifest-path`".

Reference: https://kubernetes.io/docs/admin/kubelet/

- [ ] Create a new Ignition image with above flag and save this image with version v2 at
`docker.io/cnastorage/k8s-ignition:v1`
- [ ] Test kubernetes cluster deployment for  v1.6.4 and v1.7.0-beta.0 using new ignition image.

**Comment by divyenpatel** [comment]
updated documentation: https://github.com/kubernetes/kubernetes-anywhere/pull/407
